src/solver/utils/ortools_utils.cpp

```diff
@@ -322,7 +322,9 @@ const std::map<std::string, struct OrtoolsUtils::SolverNames> OrtoolsUtils::solv
   {"sirius", {"sirius_lp", "sirius"}},
   {"coin", {"clp", "cbc"}},
   {"glpk", {"glpk_lp", "glpk"}},
-  {"scip", {"scip", "scip"}}};
+  {"scip", {"", "scip"}}, // SCIP only supports MIPs
+  {"highs", {"highs_lp", "highs"}},
+  {"pdlp", {"pdlp", ""}}}; // PDLP only supports LPs
```

Comment:
Consider using `std::optional<std::string>` instead of empty strings, it may simplify e.g function `translateSolverName`